src/cross_section/include/cross_section.h

```diff
@@ -54,6 +56,8 @@ class CrossSection {
   CrossSection Translate(glm::vec2 v);
   CrossSection Rotate(float degrees);
   CrossSection Scale(glm::vec2 s);
+  CrossSection Mirror(glm::vec2 ax, bool reverse = true);
```

Comment:
Let's merge this code into `Scale` - a negative scale should automatically mirror. This is how `Scale` works for `Manifold` too: it can invert the winding.

Reply:
Negative scaling can't mirror across arbitrary vectors without changing the scaling though can it? Or is my brain not working right now 🤔.

Reply:
Yes, but you can accomplish it with `Rotate().Scale().Rotate()`. If people feel it's important, we should add it to `Manifold` too - let's save that discussion for later.

Reply:
I'm not really a fan of that as a workaround to be honest. Is there anything to stop us adding axis mirroring to Manifold instead of removing it here? Does applying the 3d version of this formula to each point and reversing the faces of a manifold to compensate something that trips over the properties that manifold tracks that makes it a more complicated operation than it is for plain meshes?

Reply:
No, it's the same. Honestly I think you're probably right, just figured it might be better in its own PR.

Reply:
Should we follow up with another PR to add Mirror and Invert then? Or just add them to this one?

Reply:
Let's do those in a follow-up since they affect the Manifold API.

Reply:
Ok, shall I rename Rewind to Invert then?

Reply:
I think I'd rather remove it entirely in favor of the fill rule argument discussed below. I think they're basically two ways of addressing the same issue.

Reply:
Ok, sure. The norm I'd already to flip meshes if they don't agree with manifolds convention when constructing from a mesh anyway I guess (I do coming from OCADml meshes, since the winding is done such that it makes non-inverted polyhedrons for OpenSCAD), so with the addition of a mirror that does the flip, Manifolds shouldn't end up inverted unless negative scaling is performed.